handlers/base.py

```diff
@@ -29,12 +29,14 @@ async def send_welcome(message: Message):
         f"{bold('/coin')}\_{bold('address')} {italic('ADDRESS')} to display coin statistics for crypto address\n\n",
         f"{bold('/gas')} to display ETH gas prices\n\n",
         f"{bold('/trending')} to display trending coins\n\n",
-        f"{bold('/alert')} {italic('COIN')} " "\[< or >] ",
+        f"{bold('/alert')} {italic('COIN')} "
+        "\[< or >] ",
         f"{italic('PRICE')} to set an alert for when the coin reaches set price\n\n",
         f"{bold('/latest')}\_{bold('listings')} to display latest crypto listings\n\n"
         f"{bold('/restart')}\_{bold('kucoin')} to restart KuCoin bot 🤖\n\n",
         f"{bold('/register')} {italic('ADDRESS')} {italic('PRIVATE')}\_{italic('KEY')} to register to use PancakeSwap bot 🤖\n\n",
-        f"{bold('/buy')}\_{bold('coin')} {italic('ADDRESS')} {italic('BNB')}\_{italic('AMOUNT')} to buy coins on pancakeswap",
+        f"{bold('/buy')}\_{bold('coin')} {italic('ADDRESS')} {italic('BNB')}\_{italic('AMOUNT')} to buy coins on pancakeswap\n\n",
+        f"{bold('/chart')} {italic('COIN')} {italic('DAYS_AMOUNT')} to display coin chart\n\n",
```

Comment:
```python
f"{bold('/chart')} {italic('COIN')} {italic('DAYS')}\_{italic('AMOUNT')} to display coin chart\n\n",
```

Reply:
So that help text displays proper styles and avoid text craziness we've seen that I introduced previously.

Reply:
I used the emojize and text imports that were originally on base.py so that the replies would style out correctly

Reply:
Say no more